.cirrus/tasks.yml

```diff
@@ -153,6 +153,24 @@ public_scan_task:
     - .cirrus/pull.sh ${PUBLIC_IMAGE_NAME} ${tag} ${platform}
     - java -jar /usr/local/bin/wss-unified-agent.jar -c .cirrus/wss-unified-agent.config -apiKey $MEND_API_KEY -product ${WS_PRODUCTNAME} -project ${PUBLIC_IMAGE_NAME}:${tag} -wss.url ${WS_WSS_URL} -docker.scanImages true
 
+private_fossa_scan_task:
+  env:
+    platform: amd64
+    FOSSA_API_KEY: VAULT[development/team/sonarqube/kv/data/fossa data.api-key]
+    matrix:
+      - tag: $CURRENT_LTA_VERSION-datacenter-app
+      - tag: $NEXT_VERSION-datacenter-app
+  ec2_instance:
+    <<: *VM_TEMPLATE
+  setup_script:
+    - curl -H 'Cache-Control: no-cache' https://raw.githubusercontent.com/fossas/fossa-cli/master/install-latest.sh | bash
+  scan_script:
+    - echo "Scan the ${PUBLIC_IMAGE_NAME}:${tag} image"
+    - docker pull "${PUBLIC_IMAGE_NAME}:${tag}"
+    - .cirrus/pull.sh ${PUBLIC_IMAGE_NAME} ${tag} ${platform}
+    - fossa container analyze -p "docker-sonarqube" -b "${tag}"
+  depends_on: multi_arch_build
```

Comment:
This is a scan of the public image. I'd suggest we rename the task's name accordingly and remove the dependence on `multi_arch_build` as we don't need to build the image to run it.

Reply:
I was not sure if it was better to scan the STAGING or the PUBLIC image.
I am noticing now that I missed the first echo.
Moving everything to PUBLIC and dropping the dependency on multi_arch_build